README.md

```diff
@@ -462,7 +463,9 @@ With this enabled you'll find a SQL script with the name `<database-name>_Create
 The database name for the create script gets resolved in the following manner:
 1. `TargetDatabaseName`.
 1. Package name.
-> Note: the generated script also uses the resolved database name via a setvar command.
+> Note: 
+>- the generated script also uses the resolved database name via a setvar command.
+>- if `IncludeCompositeObjects` is true, the composite objects (tables, etc.) from external references are also included in the generated script   
```

Comment:
Perhaps it is worth noting here that `IncludeCompositeObjects` also defaults to true (see [here](https://github.com/rr-wfm/MSBuild.Sdk.SqlProj/blob/master/src/MSBuild.Sdk.SqlProj/Sdk/Sdk.props#L217)).

Reply:
Done